examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,149 @@
+# usage:
+#   python letter.py input.csv
+#   logs to csv's in the output directory
+
+import os
+import sys
+import lob
+import csv
+import datetime
+
+###########################################################
+# TODO: Provide the following constants
+# Your API Key, keep this secure
+lob.api_key = 'test_fc26575412e92e22a926bc96c857f375f8b'
+
+# Does the input file have headers in the first row?
+IS_HEADER = True
+
+###########################################################
+
+# Input check
+if len(sys.argv) < 2:
+    print("Please provide an input CSV file as an argument.")
+    print("usage: python letter.py <CSV_FILE>")
+    sys.exit(1)
+
+# Open input files
+input_filename = sys.argv[1]
+
+success_csv_fields = [
+    'name',
+    'id',
+    'url',
+    'address_line1',
+    'address_line2',
+    'address_city',
+    'address_state',
+    'address_zip'
+]
+errors_csv_fields = ['error']
+
+# create the output directory,
+output_dir = os.path.join('.',  'output')
+if not os.path.isdir(output_dir):
+    os.mkdir(output_dir)
+
+timestamp = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+try:
+    output_dir = os.path.join(output_dir, timestamp)
+    os.mkdir(output_dir)
+except Exception, e:
+    print('Error: %s' % str(e))
+    print('Failed to create output directory. Aborting all sends.')
+    sys.exit(1)
+
+# output csv names
+success_filename = os.path.join(output_dir, 'success.csv')
+errors_filename = os.path.join(output_dir, 'errors.csv')
+
+with open('letter.html', 'r') as html_file:
+    letter_html = html_file.read()
+
+with open(input_filename, 'r') as input, \
+        open(success_filename, 'w') as success, \
+        open(errors_filename, 'w') as errors:
+
+    # Print mode to screen
+    mode = lob.api_key.split('_')[0]
+    print('Sending letters in %s mode.' % mode.upper())
+
+    input_csv = csv.DictReader(input)
+    errors_csv_fields += input_csv.fieldnames
+
+    success_csv = csv.DictWriter(success, fieldnames=success_csv_fields)
+    errors_csv = csv.DictWriter(errors, fieldnames=errors_csv_fields)
+
+    # Create from_address
+    try:
+        from_address = lob.Address.create(
+            name='Your Name/Company',
+            address_line1='123 Test Avenue',
+            address_city='San Francisco',
+            address_state='CA',
+            address_zip='94107',
+        )
+    except Exception, e:
+        print('Error: %s' % str(e))
+        print('Failed to create from_address. Aborting all sends.')
+        sys.exit(1)
+
+    # Loop through input CSV rows
+    for idx, row in enumerate(input_csv):
+        if IS_HEADER and idx == 0:
```

Comment:
Not only does it make it unnecessary, it makes this a bug. I just realized it hasn't been sending the first row xD